examples/userapi/07_functions_on_subdomains.ipynb

```diff
@@ -0,0 +1,3027 @@
+{
```

Comment:
<p>Missing velocity damp</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
<p>Are you supposed to damp both tau and v?</p>

Reply:
Fwiw, we don't damp velocity elsewhere